### PR TITLE
Torped i BallGrabber

### PR DESCRIPTION
**Pull request by Korczak**

### CODE REVIEW

File: Assets/torped.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+
+public class torped : MonoBehaviour
+{
```
Comment (Korczak) [inline Assets/torped.cs]: Make class with upper letters Torped

File: Assets/torped.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+
+public class torped : MonoBehaviour
```
Comment (Korczak) [inline Assets/torped.cs]: Add this and other components to Robot.cs

File: Assets/ballGrapper.cs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    private void Start()
+    {
+        state = State.CANT_HOLD;
+    }
+
+
+
+
+    private void FixedUpdate()
+    {
```
[the remark this answers is not in the document]
Comment (szczoteczka) [reply Assets/ballGrapper.cs]: I think so because it uses the phisic function if i use update then the symulation will work very slow

**Review by DominikWasiolka — APPROVED** [review]

**Comment by piotrszleg** [comment]
All of the important changes were added to [Refactoring #69](https://github.com/pwrdc/TransdecEnvironment/pull/69#issue-465073551) after clearing up naming issues.

**Pull request closed without merging**